docs/library/index.rst

```diff
@@ -57,4 +88,5 @@ Contents
    util_rfifo/index
    util_wfifo/index
    util_var_fifo/index
-   common/ad_dds/index
+   axi_adxcvr/index
```

Comment:
Maybe it would be easier to follow if we have them in alphabetical order

Reply:
Agree